applications/NXmx.nxdl.xml

```diff
@@ -152,7 +152,14 @@
 						However, in the case of multiple detector elements, each element
 						needs a uniquely named NXdetector group.
 					</doc>
-					<field name="depends_on" type="NX_CHAR" /> <!-- better type for paths needed  -->
+					<field name="depends_on" type="NX_CHAR"> <!-- better type for paths needed  -->
+						<doc>
+							NeXus path to the detector positioner axis that most directly
+							supports the detector.  In the case of a multi-module detector,
+							this should be the most common axis, or ‘.’ if no axes are in
```

Comment:
The quote marks around the period look curly.  Likely to cause Sphinx problems.  Change to 

    ``.``